templates/managed/index.html

```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
         </div>
       </div>
     </div>
-    <p>
-      <a href="/engage/private-cloud-cost-analysis-webinar">Watch "Hosted private cloud infrastructure: A cost analysis" webinar</a>
-    </p>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-8">
-      <h2>
-        A frictionless cloud journey
-      </h2>
-      <p>
-        Building your private/hybrid cloud puts you at the centre of innovation. Whether you're transitioning to the cloud or already running a fully operational one, it is important to structure a business-centric IT that helps your organisation focus on innovation, cut down costs and realise the benefits of the cloud.
-      </p>
-      <p>
-        A recent Gartner <a href="https://www.gartner.com/en/publications/cio-top-priorities-leadership-vision-2021">report</a> shows that the biggest infrastructure and operations challenges facing CIOs are insufficient skilled resources and managing the technical debt. Our managed services allow you to focus on your business strategy, fill any skill gaps and offload the technical complexity of cloud operations.
-      </p>
-    </div>
-  </div>
-</section>
-
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>Human Resources Management</h2>
-      <p>Increase labor efficiency through access to qualified, experienced cloud engineers for your operations. Leverage your IT engineers’ capabilities to prepare your business for the future.</p>
-    </div>
-    <div class="col-4 u-hide--medium u-hide--small u-align--center">
-      {{
-      image (
-      url="https://assets.ubuntu.com/v1/7cacf0d8-little+men.svg",
-      alt="",
-      width="185",
-      height="145",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<div class="u-fixed-width">
-  <hr>
-</div>
-
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>Cost Efficiency</h2>
-      <p>Cut your operational costs and direct your budgets to advance your business. With the straightforward pricing model, you don’t have to predict and prove anymore.</p>
-    </div>
-    <div class="col-4 u-hide--medium u-hide--small u-align--center">
-      {{
-      image (
-      url="https://assets.ubuntu.com/v1/49761812-Budgeting.svg",
-      alt="",
-      width="149",
-      height="150",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
+<section class="u-fixed-width">
+  <hr class="p-rule">
+  <div class="p-strip is-deep">
+    <h2>
+      Case study: <a href="https://ubuntu.com/engage/kubernetes-openstack-case-study-firmus">Firmus builds a sustainable
```

Comment:
This link should be relative

Reply:
Sorry, I didn't understand this comment, can you please elaborate more?

Reply:
It should not contain that domain and be relative so local and demo navigation remains on the same platform.